specification/postgresqlhsc/resource-manager/Microsoft.DBforPostgreSQL/stable/2022-11-08/examples/RoleCreate.json

```diff
@@ -0,0 +1,56 @@
+{
+  "parameters": {
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "TestGroup",
+    "clusterName": "pgtestsvc4",
+    "roleName": "role1",
+    "api-version": "2022-11-08",
+    "parameters": {
+      "properties": {
+        "password": "p@ssword1234"
```

Comment:
>p@ssword1234 [](http://example.com/codeflow?start=21&length=12)

CredScan Test is failing due to the sample passwords mentioned in examples. I would recommend to remove these sample passwords in example, and keep these properties empty in examples.

Reply:
Fixed passwords